index.html

```diff
@@ -7,15 +7,78 @@
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
+
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
+    <link rel="stylesheet" href="css/bodystyle.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+
 </head>
 <body>
-
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header>
+        <nav>
+            <input type="checkbox" id="check">
+            <label for="check" class="menubutton">
+                <i class="fa fa-bars"></i>
+            </label>
+            <img class="logo" src="./img/karma-logo.svg" alt="logo">
+            <ul>
+                <li><a href="#" style="font-weight: bold;">Meet Karma</a></li>
+                <li><a href="#">How it Works</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <section id="hero">
+            <div>
+                <h2>Introducing Karma</h2>
+                <p>Bring WIFI with you, everywhere you go.</p>
+                <button>Learn More</button>
+            </div>
+        </section>
+
+        <section id="middle">
+            <p>Everyone needs a little Karma.</p>
+        </section>
+        
+        <section id="boxes">
+            <div>
```

Comment:
Do you need this div? It seems to have the same purpose as the `section` (they both wrap the same elements)